spec/lib/embed/viewer/geo_spec.rb

```diff
@@ -1,9 +1,37 @@
 require 'rails_helper'
 
 describe Embed::Viewer::Geo do
+  include PURLFixtures
+  let(:request) { Embed::Request.new({url: 'http://purl.stanford.edu/abc123'}) }
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.